src/style.css

```
@@ -50,18 +50,18 @@ body {
   display: flex;
 }
 
-.stars--1:first-child,
-.stars--2:nth-child(-n+2),
-.stars--3:nth-child(-n+3),
-.stars--4:nth-child(-n+4),
-.stars--5 {
+.stars {
```

```suggestion
.star {
```
this is one star, not few

---

I'll take this into consideration)